src/include/mpir_thread.h

```diff
@@ -27,6 +28,22 @@ typedef struct {
 } MPIR_Thread_info_t;
 extern MPIR_Thread_info_t MPIR_ThreadInfo;
 
+typedef struct MPIR_Thread_sync {
+    struct MPIR_Thread_sync *next;
+    struct MPIR_Thread_sync *prev;
+    int is_server;
+    int initialized;
```

Comment:
Nitpick - one boolean value is named `is_<foo>` and the other isn't. We should be consistent.

Reply:
done.